src/aod_sketch_pg_functions.c

```diff
@@ -173,8 +173,7 @@ Datum pg_aod_sketch_get_estimate_and_bounds(PG_FUNCTION_ARGS) {
 
   bytes_in = PG_GETARG_BYTEA_P(0);
   sketchptr = aod_sketch_deserialize(VARDATA(bytes_in), VARSIZE(bytes_in) - VARHDRSZ);
-  num_std_devs = PG_GETARG_INT32(1);
-  if (num_std_devs == 0) num_std_devs = 1; // default
+  num_std_devs = PG_NARGS() > 1 ? PG_GETARG_INT32(1) : 1;
```

Comment:
what exactly does `PG_NARGS()` return? This checks if the value is strictly greater than 1, but asks for value 1.

I'm blocked on reviewing this without knowing if that's reasonable.

Reply:
It returns the number of arguments. So in this case if PG_NARGS() > 1 then there are at least two arguments. Therefore PG_GETARG_XXX(1) should be valid (zero-based index here).